c++/src/kj/compat/tls.c++

```diff
@@ -160,11 +161,21 @@ public:
       }
     });
   }
+
   kj::Promise<void> accept() {
     // We are the server. Set SSL options to prefer server's cipher choice.
     SSL_set_options(ssl, SSL_OP_CIPHER_SERVER_PREFERENCE);
 
-    return sslCall([this]() { return SSL_accept(ssl); }).ignoreResult();
+    return sslCall([this]() {
+             return SSL_accept(ssl);
+           })
+        .then([](size_t ret) {
+          if (ret == 0) {
```

Comment:
I did run into it, but only because I had been seeing what I could do with BIO_read. I think it's possible but I don't think it's likely we see it in practice.